invenio/modules/pages/admin.py

```diff
@@ -60,6 +62,7 @@ class PagesAdmin(ModelView):
 
     form_args = dict(
         template_name=dict(
+            default="default.html",
```

Comment:
It used to work with lambda function or alternatively you can wrap it to a LocalProxy.